clients/privacy-center/common/i18nContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { createContext, useMemo, useState } from "react";
+import { I18n, setupI18n } from "fides-js";
+
+interface I18nContextProps {
+  currentLocale: string | undefined;
+  setCurrentLocale: (newLocale: string) => void;
+  i18nInstance: I18n;
```

Comment:
Storing i18n singleton in the context to avoid passing it down through multiple components